README.md

```diff
@@ -25,3 +25,9 @@ and solve it as we did in the lectures. Give the final $\Theta$ complexity.
 
 Describe your reasoning and the conclusion you've come to. Your reasoning is the
 most important part. Add your answer to this markdown file.
+
+The recurrence relation is T(n)=3T(n/3)+O(n). It runs in the time complexity (Θ(n log n)). This is because the function makes three recursive calls on subarrays of size n/3 while doing O(n) work.
```

Comment:
You cannot have an asymptotic expression ($O(n)$) in your recurrence relation. And how much work needs to be done to combine the partial results?